composer.json

```diff
@@ -24,10 +24,10 @@
 		"symfony/process": "^5.0||^4.0||^3.0||~2.3",
 		"aws/aws-sdk-php": "~3.2",
 		"psr/log": "~1.0",
-        "microsoft/azure-storage-blob": "^1.4"
+        "microsoft/azure-storage-blob": "^1.4",
+		"phpunit/phpunit": "7"
```

Comment:
Ono to je asi jedno ked na produkciu posielame ten build v ktorom je aj `require-dev` len ma zaujimalo preco je ot teraz tu

Reply:
phpunit sem přesunul do dev, to byla chyba, jak sem to mazal tak sem asi zapomněl hodit --dev flag. :+1: